crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -1031,7 +1034,7 @@ impl Rhombus {
     #[inline(always)]
     pub fn from_side(side: f32) -> Self {
         Self {
-            half_diagonals: Vec2::splat(side.hypot(side) / 2.0),
+            half_diagonals: Vec2::splat(ops::hypot(side, side) / 2.0),
```

Comment:
Not sure when I missed this, but this simplifies down to:
```
hypot(side, side) / 2.0 -> side * 1/sqrt(2)
```
Not the fault of this PR tho.

Reply:
You're still missing a `sqrt` on `side` I think, but otherwise yeah

Reply:
Nope, hypot first squares and then square roots.
hypot(x,x) -> sqrt(x^2 + x^2) -> sqrt(2x^2) -> sqrt(2) * sqrt(x^2) -> sqrt(2) * abs(x)
So I suppose there is a difference for negative values, however a negative side length is a nonsensical input anyway.

Reply:
I somehow forgot about the inner squares in `hypot`; you're right. Hahaha